packages/material-ui/src/Dialog/Dialog.js

```diff
@@ -225,6 +227,11 @@ const Dialog = React.forwardRef(function Dialog(inProps, ref) {
     }
   };
 
+  const ariaLabelledby = useId(ariaLabelledbyProp);
```

Comment:
What's the story for dialogs that don't render a `<DialogTitle>`? For instance https://next.material-ui.com/components/dialogs/#full-screen-dialogs. Does an `aria-labelledby` pointing to a non existing DOM node an a11y issue?

Reply:
> What's the story for dialogs that don't render a <DialogTitle>?

They are already problematic.

> Does an aria-labelledby pointing to a non existing DOM node an a11y issue?

Nope. It will be treated just like it has no aria-labelledby

Reply:
> Nope. It will be treated just like it has no aria-labelledby

Great, and we even have the optionality to have DialogContent host the label` if needed, in the future.